dali/python/nvidia/dali/plugin/mxnet.py

```diff
@@ -57,7 +60,10 @@ def feed_ndarray(dali_tensor, arr):
     ptr = ctypes.c_void_p()
     mx.base._LIB.MXNDArrayGetData(arr.handle, ctypes.byref(ptr))
     # Copy data from DALI tensor to ptr
-    dali_tensor.copy_to_external(ptr)
+    if isinstance(dali_tensor, nvidia.dali.backend.TensorGPU):
```

Comment:
I wonder how `mx.nd.zeros` works and if we won't have the same problem as in PyTorch.

Reply:
There is this call to `mx.base._LIB.MXNDArrayWaitToWrite(arr.handle)`, not sure if that'd be enough

Reply:
Not to mention PaddlePaddle.